src/components/contenedor-medicamentos/contPastis/index.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import styles from "./style.module.scss";
+import Tarjetas from "./tarjetas";
+
+class ContPastis extends React.Component{
+
+
+    render(){
+        return(

```

Comment:
Atención a los espacios entre instrucciones.
```suggestion
    render() {
        return (

```